docs/faq.rst

```diff
@@ -84,6 +84,28 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
 
 :ref:`(back to top) <topFAQ>`
 
+.. _what-is-dockstore-generative-ai-policy:
+
+In plain language, what is Dockstore's approach to generative AI?
+-----------------------------------------------------------------------
+
+In broad strokes, we are exploring the use of generative AI to enhance the workflow and tool development process.
+For full transparency, we will work following these principles:
+
+1. For content generated and displayed to the public or to other users (e.g. AI generated topic sentences), we will only train or use as prompts, data that is already available publicly.
+In other words, as an example, only after the publish button is used, will we create topic sentences that are presented to other users.
```

Comment:
Nit: instead of `after the publish button is used`, I'd suggest something like `after publishing a tool  or workflow`, to handle non-UI published entries. API publishing probably isn't that common, but .dockstore.yml publishing is.